analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -331,8 +331,7 @@ class SpdxDocumentFile(
         packages: MutableSet<Package>
```

Comment:
Commit message: I have problems understanding the message. What is meant by _dependsOnCase function_? Could this be rephrased, e.g. like "Support external document references when querying dependencies in the DEPENDENCY_OF relation by delegating to getSpdxPackageForId()."? (Not sure whether this is correct as I am not too familiar with SPDX.)

Reply:
The "Remove bug" part in the commit message is still a bit strange. Shouldn't this be something like "This fixes the bug that an exception is thrown..." ?